CHANGELOG.md

```diff
@@ -1,21 +1,31 @@
 ## Release notes
 
+### 0.13.1 -- Apr 16, 2021
+* Add `None` as an alias for `IS NULL` comparison in `dict` restrictions (#824) PR #893
+* Drop support for MySQL 5.6 since it has reached EOL PR #893
+* Bugfix - `schema.list_tables()` is not topologically sorted (#838) PR #893
+* Bugfix - Diagram part tables do not show proper class name (#882) PR #893
+* Bugfix - Error in complex restrictions (#892) PR #893
+* Bugfix - WHERE and GROUP BY clases are dropped on joins with aggregation (#898, #899) PR #893
```

Comment:
```suggestion
* Bugfix - WHERE and GROUP BY clauses are dropped on joins with aggregation (#898, #899) PR #893
```

Reply:
@dimitri-yatsenko This is just a formal PR to catch up `pre` with `master`. It should match exactly. I agree this is a simple typo but it should be addressed in `master` branch first. This change would throw off the hash.